### ISSUE TITLE
Option for TabGroups button to take directly to TabGroups View

### ISSUE THREAD

**Issue by jjspace** [Quicksaver/Tab-Groups#242]
So I love the new update and I think many of the features are very useful. However over the years I have gotten really used to quickly hitting the TagBroups button and immediately seeing my groups and then switching to a different one. The main reason I prefer this is because most of the time when I'm switching I want to go to a specific tab within a group and seeing them all in the Grid or Free Arrange format lets me see the thumbnails and quickly switch to the one I want instead of switching to the group and THEN going to the tab I want. (this is also helpful for memory because I have a lot of tabs and I have it set that they only load when I click on them again so the less tabs I have to load the better in my opinion) I do see the practicality of the drop down and will probably use it at times but I think I would prefer the other way.

so tl;dr It would be great to have an option to go straight to the TabGroup view when you hit the TabGroup button like how it used to work.

Also I think someone else mentioned it in another issue but it would be great if there was a way to shrink the button back down into the little square it used to have because that is precious space that my many tabs could be using.
Keep up the great work!

**Comment by philnolan3d** [comment]
I totally agree about the button. I spent like a half hour trying to figure out where the button went and complain that it was missing before I discovered it hidden behind this "misc" drop down. It should be just a button like it always was.

**Comment by Quicksaver** [comment]
Both of those options already exist, check the add-on's preferences for "Show the name of the active group" and "Click to open the quick access panel" in the toolbar button.

**Comment by DanielKeep** [comment]
I was just about to look for some way to forcibly roll back to the old version because I couldn't see how to get back the old behaviour.  It's *incredibly* disconcerting to have the behaviour of one of the most used buttons in my browser suddenly undergo a radical change in behaviour with no explanation on how to undo it.

**Comment by Quicksaver** [comment]
There was an explanation in the very first item of the release notes.

**Comment by jjspace** [comment]
Oh thank you, for some reason when i was looking through all the settings my brain just glossed right over those two options.

**Comment by DanielKeep** [comment]
> The quick access panel lets you quickly switch between tabs and groups without having to enter groups view ...

Neither "quick access panel" nor "groups view" mean anything because neither has been defined.  Before now, the addon only had one behaviour, so as far as I know, nothing had a name beyond "tab groups".

Also, I disabled both likely-looking checkboxes, and the button *still* doesn't behave the way it used to.  I actually thought I'd fixed this yesterday, but apparently the "free mode" (which took me a while to find because, again, it didn't have a name previously) doesn't stick or something?

Really, my major issue is that nearly everything about the button's functionality just changed overnight.

Don't get me wrong, I do *very much* appreciate that this is being kept alive.  I just get really frustrated at surprise updates that alter basic functionality without some kind of direct "I'm happy with how things are" option front and center.  I'm *only* bringing this up in the hope that if a change like this happens in the future, it's a little smoother.

**Comment by Quicksaver** [comment]
> Neither "quick access panel" nor "groups view" mean anything because neither has been defined. Before now, the addon only had one behaviour, so as far as I know, nothing had a name beyond "tab groups".

And now they do, hence why those have the "new" bullet points in the release notes. ;) It's also all explained in either the add-on's description in the homepage, or in the "how to use" page.

> Also, I disabled both likely-looking checkboxes, and the button still doesn't behave the way it used to. I actually thought I'd fixed this yesterday, but apparently the "free mode" doesn't stick or something?

It should remember the last mode you used, so that you don't need to keep switching. Could you please disable all your other add-ons and see you Tab Groups settings stick then? It sounds like another add-on may be resetting your preferences for Tab Groups.
